episodes/introduction.md

```diff
@@ -1,28 +1,246 @@
+
+You'd want to be able to **collaborate** with other people starting early on in
+the project - it'd be nice if your other team members could review your work,
+provide feedback, and contribute to parts of the analysis. Then if you have to
+disappear for a while, other people can carry the project forward.
+
+You'd want to be **open and transparent** about the data - what the inputs are,
+what the outputs are, and how you got from A to B. When you're talking to a
+potential collaborator, a new teammate, or just getting back into the project
+after a break, being able to see the core data flows can save you a lot of
+pain.
+
+You'd also want the project outcomes to be **reproducible**, so researchers in
+other labs can build on the work that you're about to do, and confirm your
+results. Maybe you end up getting a new job, leaving someone else in the lab to
+keep the project going. In your new role, you might find that you yourself want
+to build on all your old work.
+
+We'll keep these principles in mind as we work through the material in this
+course.
+
+### Welcome
+
+Welcome to Open Energy Data for All!
+
+:::: instructor
+
+Deliver this as if you are in an infomercial, and as if everyone is on board with the character you are playing.
+
+::::
+
+* Have you ever struggled with all the weird little auxiliary bits of writing research software?
+* Does it ever seem like those weird little auxiliary bits are, like, all of the work you do, and the actual interesting analysis stuff falls by the wayside?
+* Do you ever feel like there must be a better way to do your research?
+
+That's all normal. There's a lot to the research process that doesn't get covered in class, and people are usually left to learn through personal struggle. We won't be able to cover all of that material, but we've chosen a few areas to cover in our time together to help close that gap.
+
+### The research life cycle
+
+Research is cyclic by nature: we ask questions about the world, and as we investigate them, we make hypotheses, test those hypotheses, and reveal more questions along with our conclusions.
+You may have seen a diagram like this in a research methods class or as part of orientation for your research institute.
+If not, that's okay!
+You don't need to know it by heart to succeed in this course.
+
+![Project life cycle diagram. Main sequence in yellow shows research stages, from choosing a topic to publication and archiving. Central to the yellow cycle is sharing work-in-progress, with components for ensuring source data is available, making data preparation methods available, distributing tools, and making results replicable. Data sequence in grey includes finding, fetching, cleaning, debugging, and fixing problems, with a looming backdrop of dealing with scale. Scattered grey bubbles indicate connections to data processes throughout the research and collaboration components.](./fig/ep1-data-project-lifecycle.png)
```

Comment:
Alt text for diagrams: I tried to strike a balance here between "placeholder that summarizes but does not replicate contents" and "full equal-access description"

I would also accept switching to a summary and linking to a full equal-access description elsewhere.